vllm/attention/backends/xformers.py

```diff
@@ -259,6 +261,8 @@ def forward(
                     query, key, value, attn_metadata)
             else:
                 # prefix-enabled attention
+                # TODO(Hai) this triton kernel has regression issue with
+                # FP8 KVCache to handle mixed types
```

Comment:
what does this mean? accuracy issue or the kernel will break? for both, please note in the docs that prefix caching is currently not compatible with fp8

Reply:
kernel incapable therefore will break, added to docs.

Reply:
solved